test/app/test/interactive.clj

```diff
@@ -22,3 +41,19 @@
                (verify-call-times-for println 2)
                (verify-first-call-args-for println
                                            no-classes-loaded-error)))))
+
+(deftest can-execute-mc-operation
+  (let [javaparser-cus {:cus (take 2 (cus javaparser-cus-path))}
```

Comment:
as I found out yesterday, the order of the Compilation Units returned from cus vary on different computers (I had tests passing locally and breaking on Travis because of that). Is this a problem in this test? If it is, can you please refactor, if not can you please add a comment explaining that it does not matter which 2 CUs you take?

It is just that I broke the master branch yesterday because of that and now I am more cautious :D